src/yb/client/batcher.cc

```diff
@@ -312,7 +312,7 @@ void Batcher::CombineError(const InFlightOp& in_flight_op) {
 }
 
 void Batcher::LookupTabletFor(InFlightOp* op) {
-  PUSH_WAIT_STATUS(util::WaitStateCode::LookingUpTablet);
+  // PUSH_WAIT_STATUS(util::WaitStateCode::LookingUpTablet);
```

Comment:
I would also comment out the YBClient wait codes in wait_state.h to ensure that we aren't mistakenly using it elsewhere.